### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by gokhan721**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,22 +42,21 @@ Environment variables have **higher precedence** over initialization parameters.
...
-| thundra_agent_lambda_metric_disable           |  bool  |              false               |
-| thundra_agent_lambda_log_disable              |  bool  |              false               |
-| thundra_agent_trace_request_skip              |  bool  |              false               |
-| thundra_agent_trace_response_skip             |  bool  |              false               |
-| thundra_agent_lambda_timeout_margin           |  int   |               200                |
-| thundra_agent_lambda_report_rest_baseUrl      | string |     https://api.thundra.io/v1    |
-| thundra_agent_lambda_report_cloudwatch_enable |  bool  |              false               |
-
+| THUNDRA_APIKEY                                | string |                -                 |
+| THUNDRA_APPLICATION_NAME                      | string |                -                 |
```
Comment (serkan-ozal) [inline README.md:50]: except `THUNDRA_APIKEY`, all other env var should have must start with `THUNDRA_AGENT_...`

File: README.md [hunk 2]
```diff
@@ -58,18 +68,251 @@ Check out the [configuration part](https://docs.thundra.io/python/configuration-
...
+```
+
+To run on your framework please refer to [supported frameworks](#frameworks)
+
+###  Request Body
+
+Normally thundra does not get your project's requests' body. In order to see request body on tracing data, following environment variable shall be set besides api key and application name:
+
+```sh
+export THUNDRA_TRACE_REQUEST_SKIP=True
```
Comment (serkan-ozal) [inline README.md:154]: `THUNDRA_AGENT_TRACE_REQUEST_SKIP`

File: README.md [hunk 3]
```diff
@@ -105,3 +348,113 @@ logger.addHandler(handler)
...
+
+## All Environment Variables
+
+Following table shows all environment variables of [No-Code Change Tracing](#no-code-change-tracing) and their [In-Code Configuration Tracing](#in-code-configuration-tracing) equivalent.
+
+
+| Environment Variable Name                                        |  [In-Code Configuration Tracing](#in-code-configuration-tracing) Name     |
+|:-----------------------------------------------------------------|:--------------------------------------------------------------------------|
+| THUNDRA_APIKEY                                                   | thundra.apikey                                                            |
+| THUNDRA_DEBUG_ENABLE                                             | thundra.agent.debug.enable                                                |
```
Comment (serkan-ozal) [inline README.md:381]: except `THUNDRA_APIKEY`, all other env vars should have must start with `THUNDRA_AGENT_...`

File: README.md [hunk 4]
```diff
@@ -58,18 +68,251 @@ Check out the [configuration part](https://docs.thundra.io/python/configuration-
...
 
-Instead, you can [setup async monitoring](https://apm.docs.thundra.io/performance/zero-overhead-with-asynchronous-monitoring) in **2 minutes** and monitor your lambda functions with **zero overhead**!
 
 ## Log Plugin
 Log plugin is added by default, but in order to enable it, you should add `ThundraLogHandler` to your logger.
 
 You can either add it via logging.conf or add during getting logger object.
 
-#### Configurepy tes via logging.conf
+#### Configurepy test via logging.conf
```
Comment (erdogan-yesil) [inline README.md:315]: "Configurepy test via logging.conf" should be "Configure via logging.conf"

File: README.md (hunk 2 above)
Comment (gokhan721) [reply README.md]: done.

File: README.md (hunk 3 above)
Comment (gokhan721) [reply README.md]: done.

File: README.md (hunk 4 above)
Comment (gokhan721) [reply README.md]: done.

**Review by serkan-ozal — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -58,18 +68,251 @@ Check out the [configuration part](https://docs.thundra.io/python/configuration-
...
+
+    ```python
+    import thundra
+
+    thundra.configure(
+        options={
+            "config": {
+                "thundra.apikey": <your_thundra_api_key>,
+                "thundra.agent.application.name": <your_application_name>,
+                "thundra.agent.trace.integrations.django.orm.disable": True,
```
Comment (erdogan-yesil) [inline README.md:228]: for tracing django database process, this config value should be "False"

File: README.md [hunk 6]
```diff
@@ -58,18 +68,251 @@ Check out the [configuration part](https://docs.thundra.io/python/configuration-
...
+                "thundra.agent.trace.integrations.rdb.disable": True
```
Comment (erdogan-yesil) [inline README.md:229]: for tracing django database process, this config value should be "False"

File: README.md (hunk 5 above)
Comment (gokhan721) [reply README.md]: done.

File: README.md (hunk 6 above)
Comment (gokhan721) [reply README.md]: done.

**Review by erdogan-yesil — APPROVED** [review]

**Pull request merged**